onnxruntime/core/framework/feeds_fetches_manager.h

```diff
@@ -38,11 +38,17 @@ struct FeedsFetchesInfo {
   // set the ort_value_idxs for the current values in feed_names and output_names
   Status SetMLValueIdxs(const OrtValueNameIdxMap& ort_value_name_idx_map);
 
-  std::vector<std::string> feed_names;
-  std::vector<std::string> output_names;
+  FeedsFetchesInfo(FeedsFetchesInfo&& other) = default;
+  FeedsFetchesInfo& operator=(FeedsFetchesInfo&&) = default;
+
+  const std::vector<std::string>& feed_names;
+  const std::vector<std::string>& output_names;
 
```

Comment:
Do we need to have references here tying this class to the creator's lifetime, or would it have been sufficient just to have the fix to use std::move when initializing feeds_fetches_info_ in the FeedsFetchesManager ctor?